### ISSUE TITLE
Admin can ban itself

### ISSUE THREAD

**Issue by CleverFool77** [publiclab/plots2#5553]
## Description
Hi @jywarren This is one of the unique issue that I found.
While trying out the the revision history and the exploring button.
I clicked on the ban button of revision history and as I was logged in as admin. And the button ban which I clicked was of the change done by me only. So when I clicked on the button, I got the message `the instant is removed and user is banned`, Thus I banned myself.

![Screenshot from 2019-04-20 11-08-51](https://user-images.githubusercontent.com/26685258/56453470-a7848180-6360-11e9-821f-a7af2e587338.png)

**Comment by CleverFool77** [comment]
@publiclab/plots2-reviewers What's your opinion regarding this ?
Thanks !!

**Comment by siaw23** [comment]
IMHO I don't think it and admin should be able to ban themselves. There might be cases where this is done accidentally.

**Comment by gauravano** [comment]
Agree with @siaw23!

**Comment by CleverFool77** [comment]
Hi @siaw23 @gauravano I tried on second image. And I did ban myself. Then I changed my status from db.
Thanks !!

**Comment by siaw23** [comment]
OK so this behaviour is not normal I think. Definitely a user shouldn't be able to ban themselves. As to what's done when a user is banned (with their changes) I think @jywarren can chime in on this one.

**Comment by IshaGupta18** [comment]
Yes, I think so too @siaw23, let's wait for his suggestion on this one, then we can probably just add a condition to remove the ban option for the current user.

**Comment by Forchapeatl** [comment]
Please may I work on this issue ?
